### PR TITLE
Change opt in label default value to allocate

### PR DESCRIPTION
**Pull request by RamLavi**
Signed-off-by: Ram Lavi <ralavi@redhat.com>

**What this PR does / why we need it**:
we want to change the label default value to 'allocate'. We also update the documentation.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by phoracek** [comment]
/hold

This does not reflect the current reality of us allocating for all interfaces, including primary

File: README.md [hunk 1]
```diff
@@ -39,15 +39,15 @@ kubectl apply -f ./kubemacpool.yaml
 ### Opting-in to kubemacpool service
 
 On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
```
Comment (phoracek) [inline README.md:41]: Please document that it is for all secondary NAD interfaces, not for the primary one.

**Comment by RamLavi** [comment]
/unhold

changed label value to `allocate`. updated the documentation accordingly

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 This project allow to allocate mac addresses from a pool to secondary interfaces using
 [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
+It also allocates mac addresses to following primary interfaces: Multus-cni, Slirp and Masquerade.
```
Comment (phoracek) [inline README.md:7]: Multus-cni is not a primary interface. We should say we allocate for secondary interfaces. Then say that for VirtualMachines, we also allocate for the primary interface if they are using masquerade or slirp binding mechanism. (and link to kubevirt.io docs regarding these settings)

File: README.md [hunk 3]
```diff
@@ -38,16 +39,16 @@ kubectl apply -f ./kubemacpool.yaml
 
 ### Opting-in to kubemacpool service
 
-On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocateForAll` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocateForAll` - to opt in vms mac allocation in your namespace
+On releases v0.8.4 and above, kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
```
Comment (phoracek) [inline README.md:42]: We should not refer to z-stream and above. It is not clear if 0.9.0 is newer or older than that. I think we can omit the mention of version completely.

Comment (RamLavi) [reply README.md:42]: ack

**Pull request merged**